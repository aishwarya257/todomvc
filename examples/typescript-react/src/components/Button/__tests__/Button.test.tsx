import React from 'react';
import Button from '../Button';

import {render} from '@testing-library/react';

describe('testing button component', () => {
    it('renders button component', () => {
        const {container} = render(<Button className="" onClick={() => {}} />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
