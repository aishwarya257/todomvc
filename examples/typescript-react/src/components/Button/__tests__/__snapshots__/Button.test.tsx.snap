// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing button component renders button component 1`] = `
<button
  class=""
>
   
   
</button>
`;
